WeakAuras/AuraEnvironment.lua

```diff
@@ -319,12 +320,79 @@ end
 
 local FakeWeakAurasMixin = {
   blockedFunctions = {
+    -- Other addons might these, so before moving them to the Private space, we need
```

Comment:
>might these

Im not sure what you actually meant but maybe?
```suggestion
    -- Other addons might use these, so before moving them to the Private space, we need
```